ocs_ci/ocs/resources/pod.py

```diff
@@ -3070,3 +3070,22 @@ def check_ceph_cmd_execute_successfully():
 
         logger.warning(f"Failed to execute the ceph command due to the error {str(ex)}")
         return False
+
+
+def get_mon_pod_by_pvc_name(pvc_name: str):
```

Comment:
Is this function needed? It seems that it is not referenced anywhere.

Reply:
you are right. The reference to it was removed when I changed the test in one of the commits. 
there only 4.10 and 4.11 left to merge, rest where merged with this func. Considering it is not making harm, should I remove it?

Reply:
I think that it would be better to remove it if it is nowhere used.